Packs/CommonScripts/.secrets-ignore

```diff
@@ -116,4 +116,195 @@ testing@gmail.com
 a853a1b0-1ffe-4e37-d9a9-a27c6bc0bd5b@gmail.com
 cipher=ECDHE-RSA-AES128-GCM-SHA256
 07.23.05.38
-multipart/signed
\ No newline at end of file
+multipart/signed
+00:87:37:26:7f:7e:da:e8
```

Comment:
Can you check if we can remove those? As IPv6 shouldnt extract those

Reply:
Those are there due to the test data, not IPv6